src/views/catalog/wizard/tabs/overview/components/VMNameModal/VMNameModal.tsx

```diff
@@ -27,7 +27,7 @@ const VMNameModal: React.FC<HostnameModalProps> = ({ isOpen, onClose, onSubmit,
 
   return (
     <TabModal
-      headerText={t('Edit hostname')}
```

Comment:
Hm, this change seems to me unrelated to https://issues.redhat.com//browse/CNV-31216 or to the UX doc you've mentioned in the description of the PR. Also I cannot see this change in the attached screenshots. So I am confused. Is this change intentional?

Reply:
they do relate, in previous PR - https://github.com/kubevirt-ui/kubevirt-plugin/pull/1428 
I add a modal to edit the name on the review section as part of the solution discussed with UX and PM.
So yes this change is itentional.
and added missing screenshots